src/entries/popup/pages/home/Points.tsx

```diff
@@ -108,79 +188,60 @@ export function Points() {
       flexDirection="column"
       justifyContent="flex-start"
       marginTop="-20px"
-      paddingTop="80px"
+      paddingTop={featureFlags.points ? '40px' : '80px'}
       ref={ref}
       style={{ height: 336 - (testnetMode ? TESTNET_MODE_BAR_HEIGHT : 0) }}
       width="full"
     >
-      <Box paddingBottom="14px">
-        <Stack alignHorizontal="center" space="16px">
+      <Stack space="14px">
+        <Box
+          alignItems="center"
+          display="flex"
+          justifyContent="center"
+          style={{
+            transform: 'translateY(-4px)',
```

Comment:
just curious, why translateY over Bleed here?

Reply:
don't know lol i copy pasted this from the placeholder we have right now